src/test/kotlin/study/PersonTest.kt

```diff
@@ -0,0 +1,36 @@
+package study
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class PersonTest {
+    @Test
+    fun `이름 붙인 인자`() {
+        val actual = Person(name = "홍길동", age = 20, nickname = "홍")
+        assertThat(actual.name).isEqualTo("홍길동")
```

Comment:
공백도 코드를 읽는데 도움이 되니 다음에는 공백을 섞어서 표현해주시면 좋을것 같습니다.
```suggestion
        val actual = Person(name = "홍길동", age = 20, nickname = "홍")
        
        assertThat(actual.name).isEqualTo("홍길동")
```